.zombienet-tests/0002-getHeaders.js

```diff
@@ -0,0 +1,40 @@
+const polkadotApi = require("@polkadot/api")
+
+async function connect(customChainSpec, types) {
+  const substrateConnect = await import("@substrate/connect")
+  const provider = new substrateConnect.ScProvider(customChainSpec)
+  const api = new polkadotApi.ApiPromise({ provider, types })
+  return api
+}
+
+async function run(nodeName, networkInfo) {
+  const { userDefinedTypes } = networkInfo.nodesByName[nodeName]
+  const customChainSpec = require(networkInfo.chainSpecPath)
+  console.log("bootnodes", customChainSpec.bootNodes)
+
+  const api = await connect(JSON.stringify(customChainSpec), userDefinedTypes)
+  // add 20s sleep to give time to sync
+  await new Promise((resolve) => setTimeout(resolve, 20000))
```

Comment:
Same remark

Reply:
We need a slight timeout for pausing the process until Chain is initialised.
If this timeout is removed then the following error shows:
```
2022-02-03 17:22:25        RPC-CORE: subscribeNewHeads(): Header:: Chain is not initialised
2022-02-03 17:22:25             DRR: Chain is not initialised
2022-02-03 17:22:25        RPC-CORE: subscribeNewHeads(): Header:: Chain is not initialised
unhandledRejection
Error: Chain is not initialised
```

Reply:
We can use the `sleep x` command in the `feature` file instead of having in this scripts.